src/integrationTest/java/uk/gov/hmcts/reform/blobrouter/data/EnvelopeRepositoryTest.java

```diff
@@ -263,13 +265,40 @@ void should_find_envelopes_by_status() {
         assertThat(repo.find(Status.REJECTED, true)).extracting(env -> env.fileName).containsExactly("f4", "f8");
     }
 
+    @Test
+    void should_return_envelopes_count_for_container_and_time_range() {
+        //given
+        Instant fromDate = now();
+        Instant toDate = fromDate.plus(3, ChronoUnit.MINUTES);
```

Comment:
A bit weird usage.
Shall we search from the past til now instead?

Reply:
Updated the test